rasa/shared/importers/autoconfig.py

```diff
@@ -263,7 +263,7 @@ def _get_commented_out_autoconfig_lines(
     autoconfig_lines = {}
 
     for key in auto_configured_keys:
-        stream = yaml.compat.StringIO()
+        stream = ruamel.yaml.compat.StringIO()
```

Comment:
but `ruamel.yaml` is imported as `yaml` above 🤷🏻

Reply:
yeah 🤷🏻 I somewhat agree with vulture that it is weird to import a module / package by alias and then access a submodule in this package through the alias.

Reply:
can we either remove the renaming of the module or tweak vulture to disregard these? sounds like a bug of vulture

Reply:
done